aws-lambda-extension-fluent-bit/README.md

```diff
@@ -0,0 +1,87 @@
+# Fluent Bit Lambda Extensions in Go
+
+The provided code sample demonstrates how to get Fluent Bit set up for Lambda Extensions. 
+
+Note: 
+1. Supported Fluent Bit: v1.6
+2. Supported AWS Fluent Bit Outputs: CloudWatch Logs, Kinesis Firehose and Kinesis Data Streams.
+
+This extension runs a goroutine that will do the following:
+* Starts the Extension and registers to ExtensionAPI,
+* Invokes Fluent Bit binary using user-specified configurations,
+* Subscribes the extension to the Logs API
+
+## Prerequisites
+
+1. You need to compile a Fluent Bit binary. The binary should be placed inside the `fluent-bit/` directory. Here are the steps:
```

Comment:
Can we include the binary you have been testing with in this repo? That way users can run the example without this step. You can reword it to say that its to update the binary to a new version.